content/hi/kubernetes.md

```diff
@@ -0,0 +1,24 @@
+---
+title: कुबेरनेट्स (Kubernetes)
+status: completed
+category: अवधारणा
+exclude_search: true
+---
+
+## यह क्या है
+
+कुबेरनेट्स, जिसे अक्सर K8s के रूप में संक्षिप्त किया जाता है, एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेटर है जो आधुनिक इन्फ्रास्ट्रक्चर पर [कंटेनरीकृत](/hi/container/)अनुप्रयोगों के जीवन-चक्र को स्वचालित करता है। यह एक डेटा सेंटर ऑपरेटिंग सिस्टम की तरह है जो एक [वितरित सिस्टम](/distributed_systems/) में चल रहे एप्लिकेशन को प्रबंधित करता है (जैसे आपके लैपटॉप पर ओएस जो आपके ऐप्स को प्रबंधित करता है)।
+
+कुबेरनेट्स एक [क्लस्टर](/cluster/) में [नोड्स](/nodes/) में कंटेनरों को शेड्यूल करता है। यह कई बुनियादी ढांचे के निर्माण को बंडल करता है, जिसे कभी-कभी "आदिम" के रूप में संदर्भित किया जाता है, जैसे कि एक ऐप का एक उदाहरण, लोड बैलेंसर, लगातार भंडारण, और अन्य एक साथ इस तरह से कि उन्हें अनुप्रयोगों में बनाया जा सकता है।

```

Comment:
I think प्राथमिक sounds better for primitive. But I guess since this term is used as a reference, I would suggest something like -  "प्रिमिटिव (प्राथमिक)".